queries.sql

```diff
@@ -58,4 +57,15 @@ FROM animals
 GROUP BY species;
 SELECT species, AVG(escape_attempts)
 FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND  '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+COMMIT;
+
+
+SELECT * FROM animals LEFT JOIN owners ON owners.id = owners_id WHERE owners_id= 4;
+SELECT * FROM animals LEFT JOIN species ON species.id = species_id WHERE species.name = 'Pokemon';
+SELECT * FROM animals JOIN owners ON animals.owners_id = owners.id;
+SELECT species.name,COUNT(species_id) FROM animals LEFT JOIN species ON species.id = species_id GROUP BY species.name;
+SELECT * FROM animals JOIN species ON species.id = species_id WHERE species.name = 'Digimon' AND owners_id=2; 
+SELECT * FROM animals JOIN owners ON owners.id = owners_id WHERE escape_attempts < 1 AND owners.full_name='Dean Winchester';
```

Comment:
- [ ] Kindly do the same for both of these queries.

Reply:
Hi @ShadyShawkat , i resolve this issue by using the following :
<img width="887" alt="2x" src="https://github.com/TamaraDimov/Vet_clinic_database/assets/116429687/e1c42c34-bba8-4a08-ae53-37decddd36aa">